src/app/pages/wallets/my-wallets/modals/dialog-create-or-update-wallet/wallet-form-modal.component.html

```diff
@@ -0,0 +1,11 @@
+<form [formGroup]='form'>
+    <h1 mat-dialog-title>{{ data ? "Update Wallet" : "Create Wallet" }}</h1>
+    <mat-form-field appearance="fill">
+        <mat-label>Wallet Name</mat-label>
+        <input matInput [formControl]='$any(form).controls["name"]'  >
+    </mat-form-field>
+    <div mat-dialog-actions>
+        <button mat-button (click)="cancel()">Cancel</button>
+        <button mat-button  (click)="save()" cdkFocusInitial>Save</button>
```

Comment:
Between `mat-button` and `(click)` there is one additional obsolete space that can be removed